merlin/study/celeryadapter.py

```diff
@@ -223,6 +227,26 @@ def start_celery_workers(spec, steps, celery_args, just_return_command):
     local_queues = []
 
     for worker_name, worker_val in workers.items():
+        worker_machines = get_yaml_var(worker_val, "machines", None)
+        if worker_machines:
+            print("check machines = ", check_machines(worker_machines))
```

Comment:
change to logger?

Reply:
Luc, it is assigning workers to steps so that a user can run ```merlin run-workers <yaml>``` and it will only start the workers for that given machine.